src/js/player.js

```diff
@@ -2473,6 +2477,32 @@ class Player extends Component {
     return this.techGet_('readyState');
   }
 
+  /**
+   * Get a video track list
+   * @link https://html.spec.whatwg.org/multipage/embedded-content.html#videotracklist
+   *
+   * @return {VideoTrackList} thes current video track list
+   * @method videoTracks
+   */
+  videoTracks() {
+    // cannot use techGet_ directly because it checks to see whether the tech is ready.
+    // Flash is unlikely to be ready in time but videoTracks should still work.
+    return this.tech_ && this.tech_['videoTracks']();
+  }
+
+  /**
+   * Get an audio track list
+   * @link https://html.spec.whatwg.org/multipage/embedded-content.html#audiotracklist
+   *
+   * @return {AudioTrackList} thes current audio track list
+   * @method audioTracks
+   */
+  audioTracks() {
+    // cannot use techGet_ directly because it checks to see whether the tech is ready.
+    // Flash is unlikely to be ready in time but audioTracks should still work.
+    return this.tech_ && this.tech_['audioTracks']();
```

Comment:
we should be able to do something like this here:
```JavaScript
audioTracks() {
  if(!this.tech_) {
    this.audioTracks_ = this.audioTracks_ || new AudioTrackList(); 
    return this.audioTracks
  }
  return this.tech_['audioTracks']();
}
```
`this.audioTracks_` is used when we loadTech_ so it should keep the same reference if thats the only place we load tech. Does that sound right?